libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -89,7 +89,8 @@ class AP_SerialManager {
         SerialProtocol_FrSky_SPort_Passthrough = 10, // FrSky SPort Passthrough (OpenTX) protocol (X-receivers)
         SerialProtocol_Lidar360 = 11,
         SerialProtocol_Aerotenna_uLanding      = 12, // Ulanding support
-        SerialProtocol_Beacon = 13
+        SerialProtocol_Beacon = 13,
+        SerialProtocol_TRTower = 14                  // TeraRanger Tower support
```

Comment:
Is there really a need for a new SerialProtocol? How is it different from Lidar360? You already have to select what 360º Lidar equipment you are using in the Proximity library.